src/store/ViewStore.js

```diff
@@ -25,6 +26,10 @@ export default class ViewStore {
     onContextReady = () => {
         const views = createObjectFromLocalStorage('cq-views');
         if (views) { this.views = views; }
+
+        setTimeout(() => {
```

Comment:
why is this necessary?

Reply:
@brucebinary to initial the PerfectScroll after rendering the list. if I don't do this, I won't be able to make scroll on the list